lib/ember-test-helpers/test-module-for-integration.js

```diff
@@ -0,0 +1,91 @@
+import Ember from 'ember';
+import TestModule from './test-module';
+import { getResolver } from './test-resolver';
+import { getContext, setContext } from './test-context';
+
+export default TestModule.extend({
+  init: function(name, description, callbacks) {
+    this._super.call(this, name, description, callbacks);
+    this.setupSteps.push(this.setupIntegrationHelpers);
+  },
+
+  setupIntegrationHelpers: function() {
+    var self = this;
+    var context = this.context;
+    context.dispatcher = Ember.EventDispatcher.create();
+    context.dispatcher.setup({}, '#ember-testing');
+    this.actionHooks = {};
+
+    context.render = function(templateString) {
+      if (Ember.isArray(templateString)) {
+        templateString = templateString.join('');
+      }
+      self.view = Ember.View.create({
+        context: self,
+        controller: self,
+        template: Ember.Handlebars.compile(templateString),
```

Comment:
We should allow for a precompiled template to be passed in. I can't recall if `Ember.Handlebars.compile` properly handles a precompiled template...